### ISSUE TITLE
JAWS does not announce Modal when opened (Close button does not appear to have focus)

### ISSUE THREAD

**Issue by justindm234** [carbon-design-system/carbon#3807]
When a Modal is opened, it is not announced/read by JAWS and its close button does not appear focused.

## Environment

Firefox ESR 60.8.0esr
JAWS Version 2019.1906.10 ILM

## Detailed description

carbon-components@10.5.1
carbon-components-react@7.5.1

## Steps to reproduce the issue

https://codesandbox.io/s/opened-modal-not-announced-by-jaws-rj7pk

**Comment by asudoh** [comment]
Hi 👋 thank you for reporting! One workaround for now is using `<ComposedModal>` instead of `<Modal>` and set `data-modal-primary-focus` to the button (close button, primary action button, etc.) that should get focus when modal gets open.

**Comment by joshblack** [comment]
Hi @justindm234! 👋 Thanks so much for taking the time to make an issue 😄 It looks like we haven't received any updates on this and it seems like there is a workaround available with `<ComposedModal>` so am going to close this. Feel free to comment if you feel like there is something I'm missing here!
